swagger-spec/api/index.html

```diff
@@ -1,7 +1,7 @@
 {
   "swaggerVersion": "1.2",
   "apiVersion": "",
-  "basePath": "https://127.0.0.1:6443",
+  "basePath": "https://10.10.10.10:6443",
```

Comment:
This PR just syncs the swagger-spec on gh-pages with the one on HEAD.
Here is the PR that changed this on HEAD: https://github.com/kubernetes/kubernetes/pull/10713